test/agentchat/test_function_call_groupchat.py

```diff
@@ -66,12 +83,12 @@ def get_random_number():
     ):
         manager = autogen.GroupChatManager(groupchat=groupchat, llm_config=llm_config)
 
-    # pass in llm_config without functions
-    llm_config_manager = llm_config.copy()
-    del llm_config_manager["functions"]
-    manager = autogen.GroupChatManager(groupchat=groupchat, llm_config=llm_config_manager)
+    manager = autogen.GroupChatManager(groupchat=groupchat, llm_config=llm_config_no_function)
 
-    user_proxy.initiate_chat(manager, message="Let's start the game!")
+    if sync:
```

Comment:
there is no check that the function was actually called. It would make sense to call a mock inside the random number function and check if it was called at least two times. This way we would catch bugs when calling async function without await and alike